lua/cokeline/config.lua

```diff
@@ -9,7 +9,13 @@ local hlgroups = lazy("cokeline.hlgroups")
 local insert = table.insert
 
 local echo = vim.api.nvim_echo
-local islist = vim.tbl_islist
+local islist = function()
```

Comment:
Hmm, maybe we could just do this check once, like `islist = vim.islist or vim.tbl_islist`?

Reply:
yeah im happy with that